objects/conversion.go

```diff
@@ -138,3 +139,123 @@ func ToByteSlice(o Object) (v []byte, ok bool) {
 	//ok = false
 	return
 }
+
+// ToMap will try to convert object o to map[string]interface value.
+func ToMap(o Object) (v map[string]interface{}, ok bool) {
+	v = make(map[string]interface{})
+	switch o := o.(type) {
+	case *Map:
+		for key, value := range o.Value {
+			v[key] = objectToInterface(value)
+		}
+		ok = true
+	}
+
+	//ok = false
+	return
+}
+
+// ToSlice will try to convert object o to []interface{} value.
+func ToSlice(o Object) (v []interface{}, ok bool) {
+	switch o := o.(type) {
+	case *Array:
+		for _, value := range o.Value {
+			v = append(v, objectToInterface(value))
+		}
+		ok = true
+	}
+
+	//ok = false
+	return
+}
+
+// objectToInterface attempts to convert an object o to an interface{} value
+func objectToInterface(o Object) (res interface{}) {
+	switch o := o.(type) {
+	case *Int:
+		res, _ = ToInt(o)
+	case *String:
+		res, _ = ToString(o)
+	case *Float:
+		res, _ = ToFloat64(o)
+	case *Bool:
+		res, _ = ToBool(o)
+	case *Char:
+		res, _ = ToRune(o)
+	case *Bytes:
+		res, _ = ToByteSlice(o)
+	case *Array:
+		values := o.Value
+		resSlice := make([]interface{}, len(values))
+		for i, v := range values {
+			nestedRes := objectToInterface(v)
+			resSlice[i] = nestedRes
+		}
+		res = resSlice
+	case *Map:
+		resMap := make(map[string]interface{})
+		for key, v := range o.Value {
+			nestedRes := objectToInterface(v)
+			resMap[key] = nestedRes
+		}
+		res = resMap
+	case Object:
+		return o
+	}
+	return res
+}
+
+// interfaceToObject will attempt to convert an interface{} v to a Tengo Object. Copy from script/conversion.go
+func interfaceToObject(v interface{}) (Object, error) {
```

Comment:
If this is an exact copy from `script/conversion.go`, let's make this a public function, like `objects.FromInterface(interface{})` and have script-side code to call this function.